src/Microsoft.Diagnostics.Monitoring.WebApi/Utilities/StackUtilities.cs

```diff
@@ -11,6 +11,13 @@
 
 namespace Microsoft.Diagnostics.Monitoring.WebApi
 {
+    internal enum StackFormat
```

Comment:
Can `CallStackFormat` from the options be used rather than making a copy here and having to convert between the two?

Reply:
All of our collection rule options are currently defined as part of dotnet-monitor.